### DISCUSSION TITLE
testcase output error

### DISCUSSION THREAD

**Discussion by sanjaync** [NASA-LIS/LISF#1591] [Q&A] [answered]
Hello sir, I have ran testcase shown in the website walkthrough. all steps i was able to run but final output of graph was not matching . please do help with this .i have attached graph pic here
![fort_cobb_output](https://github.com/user-attachments/assets/2b86f195-cd45-45a6-b737-c22a2d6b3f39)

**Comment by dmocko** [comment]
Hi @sanjaync 

Please check ALL of your lislog files for both runs (especially the OL simulation).  There may have been a problem when you ran them,
and the lislog files might indicate why.

**Reply by sanjaync** [reply, marked as answer]
thanks @dmocko . as you mentioned ,It was issue with OL output thanks for your time.
